packages/blade/src/components/Tooltip/types.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { Side, UseFloatingOptions } from '@floating-ui/react';
+import type { CSSProperties } from 'react';
+import type { Platform } from '~utils';
+import type { BaseBoxProps } from '~components/Box/BaseBox';
+
+type TooltipProps = {
+  content: string;
+  placement?: Exclude<
+    UseFloatingOptions['placement'],
+    'left-end' | 'left-start' | 'right-end' | 'right-start'
+  >;
+  children: React.ReactElement;
+  onOpenChange?: ({ isOpen }: { isOpen: boolean }) => void;
+};
+
+type TooltipTriggerProps = {
```

Comment:
I feel these shall rather be named as some common event handlers and we can create a type for them and then use it at all the places and not scope it just to TooltipTriggerProps?

Reply:
That will work too, can rename this to `BladeCommonEvents` tho are we saying we will also expose other events here too in future?